scripts/draft-translate.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import click
+
+from googletrans import Translator  # https://github.com/ssut/py-googletrans
+
+SKIP_PREFIXES = [
+    "<!--",
+    "## {{% heading",
+    "{{< feature-state",
+    "{{< comment",
+    "{{< /comment",
+    "{{< glossary_definition",
+    "{{< note",
+    "{{< /note",
+    "{{< caution",
+    "{{< /caution",
+    "{{< table",
+    "{{< /table",
+    "{{% thirdparty",
+    "{{< codenew",
```

Comment:
Maybe we can sort it in alphabetical or logical order.

Reply:
sorted with alphabetical order